docs/src/guides/typo3/deploy/customize.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "Customize TYPO3 for Platform.sh"
+sidebarTitle: "Customize"
+weight: -90
+toc: false
+description: |
+    Add some helpful dependencies, and modify your TYPO3 site to read from a Platform.sh environment.
+---
+
+Now that you've added Platform.sh's configuration files, you will need to make a few small changes to the application itself. There are a few helpful dependencies to add, and your application will need a way to connect to the `relationships` you've defined for MariaDB and Redis by accessing some Platform.sh environment variables from within your TYPO3 configuration.
+
+## Dependencies
+
+### TYPO3 CMS's `web-dir`
+
+Platform.sh recommends serving TYPO3 from it's default subdirectory `public`. `public` can be seen already throughout your `.platform.app.yaml` file in `web.locations.root`, `mounts`, and within your `build` and `deploy` hooks. You will need to assign `public` to the `cms.web-dir` attribute in your `composer.json` file, and it's a good idea to `update` dependencies once you have done so:
+
+```bash
+$ composer config extra.typo3/cms.web-dir public && composer update --no-scripts
+```
+
+### Platform.sh Configuration Reader library for PHP
+
+TYPO3, like all applications on Platform.sh, will read information about it's environment through a collection of pre-defined environment variables. Some of them are straight forward and highly visible (i.e. `$PLATFORM_BRANCH=platformify`), while others are base64 encoded JSON objects that contain nested detailed information about that environment (i.e. `$PLATFORM_RELATIONSHIPS`).
+
+In order to make these variables, and any other you set in the future, easier to read and use within TYPO3 it's recommended that you install Platform.sh's [Configuration Reader library for PHP](https://github.com/platformsh/config-reader-php). It contains a set of common functions for interacting with the environment within a PHP application. 
+
+```bash
+$ composer require platformsh/config-reader
+```
+
+### Avoiding deadlock with the Local Page Error Handler
+
+A default TYPO3 installation has a risk of deadlocks when run on low-worker PHP-FPM configurations. Specifically, TYPO3 handles 403 and 404 error pages by issuing a full HTTP request back to itself with no timeout, which can lead to process starvation. There are two required steps to avoid this problem.
+
+First, you will need to install the [Local Page Error Handler](https://extensions.typo3.org/extension/pxa_lpeh/) extension for TYPO with the command:
+
+```bash
+$ composer require pixelant/pxa-lpeh
```

Comment:
Given that the user has a working TYPO3 instance already, you can enable the extension at this point too.

This is copied from the extension manual:

Enable the extension in the Admin tools > Extensions module or run `vendor/bin/typo3 extension:activate pxa_lpeh`